x-pack/plugin/eql/build.gradle

```diff
@@ -41,6 +41,8 @@ dependencies {
   testCompile project(path: ':modules:reindex', configuration: 'runtime')
   testCompile project(path: ':modules:parent-join', configuration: 'runtime')
   testCompile project(path: ':modules:analysis-common', configuration: 'runtime')
+
+  compile 'io.ous:jtoml:2.0.0'
```

Comment:
I'm not sure if this belongs here too or just in `eql/src/test/build.gradle`. I had some trouble getting it to compile otherwise.